packages/query/src/block-processor.ts

```diff
@@ -308,8 +308,26 @@ export class BlockProcessor implements BlockProcessorInterface {
         await this.handleEpochTransition(compactBlock.height, latestKnownBlockHeight);
       }
 
-      if (globalThis.ASSERT_ROOT_VALID) {
-        await this.assertRootValid(compactBlock.height);
+      if (globalThis.__ASSERT_ROOT__) {
```

Comment:
The body of this function is quite large as is. Think the previous helper function is more readable.

Reply:
reverted